pymeasure/instruments/sorensen/__init__.py

```diff
@@ -0,0 +1,25 @@
+#
```

Comment:
as this is a new manufacturer, we also need to import this in https://github.com/ralph-group/pymeasure/blob/master/pymeasure/instruments/__init__.py